mobilitypath_visualizer/src/mobilitypath_visualizer.cpp

```diff
@@ -158,16 +206,18 @@ namespace mobilitypath_visualizer {
         for (size_t i = 1; i < count; i++) // start id = 1 to comply with arrow marker type
         {
             marker.id = i;
-            marker.header.stamp += ros::Duration(0.1);
+            rclcpp::Time marker_cur_time(marker.header.stamp);
+            rclcpp::Time updated_time =  marker_cur_time + rclcpp::Duration(0.1 * 1e9);
```

Comment:
It might be helpful to include a comment here that `rclcpp::Duration(0.1 * 1e9)` is 0.1 seconds.